### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewExtensions_Keyboard:Tooltip is not accessible through keyboard.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52617]
**Environment Details**:
VSCode Version : 1.24.1
OS Version : WIN10
**Additional Details:**
Keyboard,MAS2.1.1
**Repro Steps:**
1. Launch VS Code.
2. Navigate to View(Alt+V) menu option and select "Extensions" from dropdown.
3. Navigate to Clear Extension input button,More button and Install button.
4. Check whether Tooltip is accessible by keyboard.

**Actual:**
Tooltip for the (More button,Clear Extension input button and Install button) is not accessible through keyboard.
Same issue observed in MAC also.
**Expected:**
Tooltip for the (More button,Clear Extension input button and Install button) should accessible through keyboard.
**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](url)

**User Impact:**
Keyboard user unable to know more  information about the button.
**MAS Reference:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}](url)

Does this issue occur when all extensions are disabled?: Yes
**Attachment For Reference**

[[Accessibility]A11y_VSCode_ViewExtension_Keyboard_Tooltip.pptx](https://github.com/Microsoft/vscode/files/2123907/Accessibility.A11y_VSCode_ViewExtension_Keyboard_Tooltip.pptx)
![accessibility a11y_vscode_viewextension_keyboard](https://user-images.githubusercontent.com/39204909/41724048-4a4301be-758a-11e8-8725-0a2f1f6d04f8.png)

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/52016
